### DISCUSSION TITLE
Fire event listener directly after app launch

### DISCUSSION THREAD

**Discussion by robingenz** [ionic-team/capacitor#5877] [Q&A] [answered]
This question is related to https://github.com/capawesome-team/capacitor-firebase/issues/140

I use the Firebase Android/iOS SDK in my plugin. These fire an event at app startup, which my plugin passes to the webview. Unfortunately, the event listeners are not yet registered at this time, so the app misses the event at startup. 

My question: Can the plugin get notified when the app is ready? The plugin `load` method fires a bit too early in my tests.

**Comment by jcesarmobile** [comment, marked as answer]
There is a third param on notifyListeners called retainUntilConsumed, if you set it to true it will keep the call until the js is set to listen to it
